### PR TITLE
[INT2-113] Add yfi 

### PR DESCRIPTION
**Pull request by CRIPU**
about yfi thanks!

### CODE REVIEW

File: governance/yfi.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+**トークンのコントラクトアドレス:** [0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e](https://etherscan.io/address/0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e)
+
+**供給量:** 36,666 / 36,666 トークンがミント済
+
+## YFIの進化
+
+YFIトークンの保有者の自立性、そしてトークン自体がどのようにYearnに関わってくるかは、ダイナミックなものを意味します。Yearnの機能のすべてがコミュニティによっていつでも変更可能であり、すでに複数の事例があります。
+YFIが台頭した週から、排出量、供給量、トークン保有者の権限をめぐって広範な議論が行われました。
+
```
Comment (DarkGhost7) [inline governance/yfi.md:15]: at line 16 should be this, was it translated?
```From the week of YFI's emergence, there was extensive discussion around emissions, supply and powers of token holders.```

File: governance/yfi.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+### YFIのミント
+
+YFI保有者の投票により、6,666個のYFIがミントされました。新たにミントされたYFIの1/3は主要なコントリビューターに帰属し、2/3はトレジャリーに送られ、ガバナンスによって展開・管理されることになりました。
+
+[YIP-57](https://snapshot.org/#/yearn/proposal/QmX8oYTSkaXSARYZn7RuQzUufW9bVVQtwJ3zxurWrquS9a)が通過した後、1日ですべてのトークンが鋳造されました。 
+
+### ガバナンス2.0における権限の定義
+
+[YIP-61](https://snapshot.org/#/ybaby.eth/proposal/QmSMyYeKrRpnA7Xn56o2NtbCUzxmhzCupL7LxMA1reXxq4)に記載されているガバナンス 2.0 では、YFIトークン保有者の責任が、Yearn Improvement Proposals（YIPs）の作成と投票から、Yearn Delegation Proposals（YDPs）および Yearn Signaling Proposals（YSPs）の作成と投票に拡大されました。
```
Comment (DarkGhost7) [inline governance/yfi.md:40]: ```Yearn Improvement Proposals（YIPs）の作成と投票から、Yearn Delegation Proposals（YDPs）および Yearn Signaling Proposals```
Translate 
Improvement proposal, delegation proposals, signaling proposals

**Review by DarkGhost7 — CHANGES_REQUESTED** [review]

File: governance/yfi.md (hunk 1 above)
Comment (CRIPU) [reply governance/yfi.md:15]: From the week of YFI's emergence, there was extensive discussion around emissions, supply and powers of token holders.
=
「YFIが台頭した週から、排出量、供給量、トークン保有者の権限をめぐって広範な議論が行われました。」 at line 14!

File: governance/yfi.md (hunk 2 above)
Comment (CRIPU) [reply governance/yfi.md]: Yearn改善プロポーザル（YIPs）の作成と投票から、Yearnデリゲーションプロポーザル（YDPs）およびYearnシグナリングプロポーザル（YSPs）
I translated!
I search about Delegation and Signaling, in Japan, they are written in katakana as a business term

**Pull request merged**